agents/heuristics.py

```diff
@@ -0,0 +1,60 @@
+from arguments import get_arguments
+from state_encodings import ENCODING_SCHEMES
+from overcooked_ai_py.mdp.overcooked_mdp import OvercookedState, OvercookedGridworld, Direction, Action
+from overcooked_ai_py.planning.planners import MotionPlanner
+
+from copy import deepcopy
+import json
+import numpy as np
+import os
+import pandas as pd
+from pathlib import Path
+import pickle
+from torch.utils.data import Dataset, DataLoader
+from tqdm import tqdm
+
+class Heuristic():
+    def __init__(self, grid, state, mlam):
+        self.state = state
+        self.players_pos_or_list = self.state.players_pos_and_or
+        self.grid = grid
+        self.terrain_mtx = self.grid.terrain_mtx
+        self.planner = mlam
+    def heuristic_1(self):
```

Comment:
please give this a better name and provide a doc string explaining the logic in english.